ai-agent/server.py

```
@@ -0,0 +1,121 @@
+        )
+    )
+    
+    active_agents[call_id] = task
+    
+    # Clean up when task completes
+    def cleanup_task(call_id: str):
+        if call_id in active_agents:
+            del active_agents[call_id]
+            logger.info(f"Cleaned up agent task for call {call_id}")
+    
+    task.add_done_callback(lambda t: cleanup_task(call_id))
```

Ruff (0.14.13)

Unused lambda argument: `t`

(ARG005)